Polyhedron/demo/Polyhedron/Plugins/PMP/Distance_plugin.cpp

```diff
@@ -294,6 +294,8 @@ class Scene_distance_polyhedron_item: public Scene_item
       double hausdorff = compute_distances(*poly_B,
                                            sorted_points,
                                            distances);
+      if(hausdorff == 0)
+        hausdorff++;
```

Comment:
Could you be more precise? What is surprizing is that you have no way to distinguish whether the distance is 1 or 0 after the increment. Why is it not important?